docs/content/en/docs/roadmap/_index.md

```diff
@@ -7,9 +7,11 @@ description: K8ssandra roadmap ideas for community consideration.
 
 K8ssandra today is deployed as an entire stack. This open-source technology currently assumes your deployment uses the entire stack. Trading out certain components for others is not supported at this time. As part of the roadmap, one goal is to support a la carte composition of components.
 
-The following additional ideas are not yet in priority order:
+The roadmap is currently tracked in GitHub on a [project board](https://github.com/orgs/k8ssandra/projects/6).
 
-* Preconfigured alerts for metrics
+The following is a previous version of the roadmap with ideas in no particular order:
```

Comment:
Is there a reason we're keeping this page if everything is going to be on the roadmap board?

Reply:
I was thinking we'd leave the current page as a breadcrumb with forwarding address to new location for some short amount of time